lago/prefix.py

```diff
@@ -1182,15 +1182,23 @@ def _copy_deploy_scripts_for_hosts(self, domains):
             for host_name, host_spec in domains.iteritems():
                 host_metadata = host_spec.get('metadata', {})
                 deploy_scripts = self._get_scripts(host_metadata)
-                new_scripts = self._copy_delpoy_scripts(deploy_scripts)
+                if host_metadata.get('custom-xml', None):
+                    host_metadata['custom-xml'] = os.path.expanduser(
+                        os.path.expandvars(
+                            self._copy_deploy_scripts(
+                                [host_metadata['custom-xml']]
+                            )[0]
+                        )
+                    )
+                new_scripts = self._copy_deploy_scripts(deploy_scripts)
```

Comment:
This will throw an exception that might be a bit confusing like 'Script mycustom.xml does not exist', maybe we can do something about that? (just asking if you have a better idea, I'll think about it too)